### PR TITLE
OCPBUGS#2194: Removing PAO section should be under E2E TALM procedure

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11, 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-2194

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information: https://51474--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#talm-PAO-update_ztp-deploying-disconnected

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-needed
/label peer-review-in-progress

**Comment by tmalove** [comment]
@amolnar-rh  What section moved in your netlify preview? I want to make sure that the offset is rendering properly. Thanks!

**Comment by amolnar-rh** [comment]
@tmalove It's the `Removing Performance Addon Operator subscriptions from deployed clusters` section

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done
/LGTM

**Comment by amolnar-rh** [comment]
Merge to 4.11 and 4.12 and if it looks good, could you please remove the `jira/invalid-bug` label?

File: scalability_and_performance/ztp-deploying-disconnected.adoc [hunk 1]
```diff
@@ -250,11 +250,11 @@ include::modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc[level
...
+
+include::modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP].
-
-include::modules/cnf-topology-aware-lifecycle-manager-operator-and-platform-update.adoc[leveloffset=+2]
+* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP].
```
Comment (jeana-redhat) [inline scalability_and_performance/ztp-deploying-disconnected.adoc:260]:
```suggestion
* xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#ztp-roll-out-the-configuration-changes_ztp-deploying-disconnected[Upgrading GitOps ZTP]
```

**Review by jeana-redhat — COMMENTED** [review]
Hey, just see one stray `.` to remove. I am not sure how the jira bug label automation works - if I remove it will it not just add it back?

**Comment by jeana-redhat** [comment]
Ok, I have renamed the PR to work around the jira automation failure here (per #49617) and will remove the label

File: scalability_and_performance/ztp-deploying-disconnected.adoc (hunk 1 above)
Comment (amolnar-rh) [reply scalability_and_performance/ztp-deploying-disconnected.adoc]: @jeana-redhat Removed the period. To answer your question, as long as I don't update the PR, it shouldn't be a problem! 
Now reading [your other comment](https://github.com/openshift/openshift-docs/pull/51474#issuecomment-1275055918), I realize someone had already mentioned this to me, probably Kathryn... I'll keep this in mind now!

**Pull request merged**